### ISSUE TITLE
Use already existing google-managed-services range for private sql access

### ISSUE THREAD

**Issue by shazzzzam** [forseti-security/terraform-google-forseti#506]
We use a shared VPC which already has a google-managed-services range peered to the VPC.
When using `cloudsql_private= true` the terraform wants to create and peer a new range.

See: 
https://github.com/forseti-security/terraform-google-forseti/blob/1c1c1ea22335a369b2349bb8f8851ecf85bc6dbe/modules/cloudsql/main.tf#L43
https://github.com/forseti-security/terraform-google-forseti/blob/1c1c1ea22335a369b2349bb8f8851ecf85bc6dbe/modules/cloudsql/main.tf#L54

Instead of creating these resources it makes sense for it to use the already existing range.

**Comment by shazzzzam** [comment]
My suggestion is to remove the above mentioned resources. According to the documentation gcp will automatically create a range for you if you haven't already created one manually.

If someone wants to create the range manually they will be free to do so outside of this module.

**Comment by rajcheval** [comment]
I ran into a related issue. We were not using shared VPC but an existing VPC.
We want to use private Cloud SQL.
Networking is configured by a different team they allocated  IP address range for us.
They also enabled service networking.
We are setting the variable "cloudsql_private" to true
Setting enable_service_networking to false
The cloud SQL module tries to allocate global address. It is only checking for cloudsq_private and not checking enable_service networking.
https://github.com/forseti-security/terraform-google-forseti/blob/master/modules/cloudsql/main.tf#L51

If the code in line 51 was changed to this it will allow us to use existing service networking and global address range.
 count                   = var.enable_service_networking && var.cloudsql_private ? 1 : 0

Finally the size of global address range is hard coded to /16. We need the ability to use much smaller range.
